src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,84 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
+            <ul class="mint-dropdown__list">
+                <li>
+                    <a class="mint-dropdown__element" href="#">Option 1</a>
```

Comment:
I am wondering whether we can make this dropdown a container in this case. 
You will be able to use `mint-link` as element. And for a standard dropdown you can use just text block/label for items